app/src/main/java/com/yeonae/chamelezone/network/api/RetrofitConnection.kt

```diff
@@ -0,0 +1,13 @@
+package com.yeonae.chamelezone.network.api
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class RetrofitConnection {
```

Comment:
```suggestion
object RetrofitConnection {
```
singleton으로 사용하는게 좋지않을까요?

Reply:
수정했습니당